src/JwksEndpoint.php

```diff
@@ -48,8 +48,13 @@ public function getPublicJwks()
         return ['keys' => $jwks];
     }
 
+    /**
+     * @deprecated
```

Comment:
It's really just a bad idea in general to `echo` anything out in a modern php app. If someone needs to echo out the json payload, it should be easy enough for them to just write the code on line 58 themselves in their app.